auth/authenticators/iam-authenticator.ts

```diff
@@ -33,33 +54,42 @@ export class IamAuthenticator extends TokenRequestBasedAuthenticator {
   private clientSecret: string;
 
   /**
-   * IAM Authenticator Class
    *
-   * Handles the IAM authentication pattern.
+   * Create a new [[IamAuthenticator]] instance.
    *
-   * @param {Object} options
-   * @constructor
+   * @param {object} options Configuration options for IAM authentication.
+   * @param {boolean} options.disableSslVerification A flag that indicates
+   *   whether verification of the token server's SSL certificate should be
+   *   disabled or not
+   * @param {string} options.url for HTTP token requests.
+   * @param {object<string, string>} options.headers to be sent with every
+   * @param {string} options.apikey The IAM api key.
+   * @param {string=} options.clientId The client_id and client_secret fields are used to form a "basic"
```

Comment:
What does the `=` sign mean here?

Reply:
jsdoc has two styles for marking items as optional and I wasn't being consistent. Fixing it now. The `=` means optional according to 1 variation of jsdoc.